packages/forms-web-app/src/pages/projects/section-51/advice-detail/controller.js

```diff
@@ -9,7 +9,8 @@ const getSection51AdviceDetailController = async (req, res, next) => {
 		const { params } = req;
 		const { locals } = res;
 
-		const adviceDetailData = await getAdviceDetailData(params.id);
+		const caseReference = locals?.caseRef;
```

Comment:
Or you could get it from the params as id is already being used from there

Reply:
I updated to destructure from `locals`. It wont be in `req` anymore. Previously I was adding to `req.query` but removed that since locals had it and this way we don't need to update any links.